From 63d658772492eaf7a1873939735ba8f3e9366ed5 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <idf2@illinois.edu>
Date: Tue, 8 Nov 2022 13:34:20 -0600
Subject: [PATCH 2/3] Always copy on windows

The conda package build machine may have permissions to
create symlinks, but conda doesn't handle symlinks on windows
properly
---
 llvm/cmake/modules/LLVMInstallSymlink.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/llvm/cmake/modules/LLVMInstallSymlink.cmake b/llvm/cmake/modules/LLVMInstallSymlink.cmake
index 274cef64b3e7..9ac4ca35e7af 100644
--- a/llvm/cmake/modules/LLVMInstallSymlink.cmake
+++ b/llvm/cmake/modules/LLVMInstallSymlink.cmake
@@ -16,10 +16,11 @@ function(install_symlink name target outdir)
 
   message(STATUS "Creating ${name}")
 
+  if(NOT CMAKE_HOST_WIN32)
   execute_process(
     COMMAND "${CMAKE_COMMAND}" -E create_symlink "${target}" "${name}"
     WORKING_DIRECTORY "${outdir}" ERROR_VARIABLE has_err)
-  if(CMAKE_HOST_WIN32 AND has_err)
+  else()
     execute_process(
       COMMAND "${CMAKE_COMMAND}" -E copy "${target}" "${name}"
       WORKING_DIRECTORY "${outdir}")
